pkgs/development/tools/misc/argbash/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, autoconf }:
+
+stdenv.mkDerivation rec {
+  pname = "argbash";
+
+  version = "2.8.1";
+
+  src = fetchFromGitHub {
+    owner = "matejak";
+    repo = "argbash";
+    rev = "${version}";
+    sha256 = "0jzfr2w5pza33mvw6hq6rm1vjgrfy95dp4421403jnji3ziwgf0w";
+  };
+
+  sourceRoot = "${src}/resources";
```

Comment:
minor: i think a relative path is sufficient

Reply:
Tried `./resources` and `resources` and it didn't work.  Seems like `${src}` is required?